client_plugin/drivers/shared/etcdops.go

```diff
@@ -33,6 +36,11 @@ const (
 	etcdScheme               = "http://"
 	etcdClusterStateNew      = "new"
 	etcdClusterStateExisting = "existing"
+	kvStoreRequestTimeout    = 5
```

Comment:
How about change this to 
`kvStoreRequestTimeout = 5*time.Second`
And replace all the following `kvStoreRequestTimeout*time.Second` to `kvStoreRequestTimeout`